packages/commonwealth/server/models/comment.ts

```diff
@@ -83,6 +83,11 @@ export default (
   );
 
   Comment.associate = (models) => {
+    models.Comment.belongsTo(models.Thread, {
```

Comment:
This association is actually not reflected in the database schema. We should not include it unless we're prepared to apply the foreign key constraint following the root id PR.